### DISCUSSION TITLE
Discord

### DISCUSSION THREAD

**Discussion by AndersNilsson19** [valinet/ExplorerPatcher#1926] [General]
Hello! Do you have a discord or wire group? Should be good so it will be easy to chatt, your software crash all the time since yesterday, have uninstalled and installed the new, but same, guess it was a win 11 update that made it crash, and "there is a update" comes up all the time, so no i cant run it at all, i uninstalled :///

**Comment by Poopooracoocoo** [comment]
Honestly will a Discord server attract any contributors or will it just become another place to flood and ping the maintainer?

**Reply by Amrsatrio** [reply]
I hope by having a Discord we can communicate better when resolving weird issues and when discussing about (mostly stupid) changes Microsoft makes in their binaries.

**Comment by Poopooracoocoo** [comment]
You shouldn't see crashes anymore. And the community could create a Discord server.

**Comment by Amrsatrio** [comment]
So it begins. https://discord.gg/gsPcfqHTD2
